gitops/kcp/gateway/kustomization.yaml

```diff
@@ -4,3 +4,5 @@ resources:
   - haproxy-deployment.yaml
   - haproxy-ingress.yaml
   - haproxy-service.yaml
+
+namespace: haproxy
```

Comment:
Checkov knows nothing about kcp. We will specify a specific namespace if we decide to deploy the gateway in the same workspace as other components.Othewise it is meaningless